deploy/helm_charts/dc_website/values.yaml

```diff
@@ -145,3 +145,8 @@ cronTesting:
   # deployment runtime.
   webApiRoot:
   nodejsRoot:
+  # The domain to use for screenshot tests. Screenshot tests will only run if
+  # this is set.
+  screenshotDomain:
+  # To disable sanity tests, set this value to 1.
+  disableSanity: 0
```

Comment:
this can be a boolean?

Reply:
This value is used in the configMap and the configMap values are always strings so checking for "0" or "1" in the script that reads these values felt more straightforward than figuring out if the string version of the boolean will be "True" or "true" or what. I can update if you feel strongly though

Reply:
Would still vote for using "true" and also name it as "enableSanity". This also better aligns with "cronTesting.enabled" field.